packages/protvista-variation-adapter/src/protvista-variation-adapter.js

```diff
@@ -1,11 +1,54 @@
 import groupBy from "lodash-es/groupBy";
+import flow from 'lodash-es/flow';
+import flatten from 'lodash-es/flatten';
+
 import ProtvistaUniprotEntryAdapter from "protvista-uniprot-entry-adapter";
 import getColor from "./variantColour";
+import filters, { getFilter } from './filters';
+
+
 export default class ProtvistaVariationAdapter extends ProtvistaUniprotEntryAdapter {
   constructor() {
     super();
   }
 
+  static get observedAttributes() {
+    return ['activefilters'];
+  }
+
+  get isManaged() {
+    return true;
+  }
+
+  attributeChangedCallback(name, oldValue, newValue) {
+    if (oldValue !== newValue) {
+      if (name !== 'activefilters') {
+        return;
+      }
+      const { sequence, variants } = this._adaptedData;
+      newValue = newValue.trim();
+      if (!newValue) {
+        this._fireEvent('load', { payload: {sequence, variants: []} });
+        return;
+      }
+      const filteredVariants = newValue
+        .split(',')
+        .map(name => getFilter(name))
+        .map(f => f.applyFilter(variants))
```

Comment:
The issue here is that this provides the union of all filters and therefore results in some overlaps (duplications in the resulting data). 
For instance when it first loads (no filters enabled) it returns 385 items, but if I click on "Likely disease" the number jumps up to 770.

Reply:
Done, I am taking `accession` as key for uniqueness, please check if that is correct.